src/react-components/ory/helpers/node.tsx

```diff
@@ -355,6 +355,8 @@ export const Node = ({
             defaultValue={attrs.value as string | number | string[]}
             required={attrs.required}
             disabled={attrs.disabled}
+            pattern={attrs.pattern}
+            fullWidth
```

Comment:
I don't believe `fullWidth` exists anymore on the inputfield.

Reply:
https://github.com/ory/elements/blob/main/src/react-components/input-field.tsx#L14-L22C2